jetstream/pull.go

```diff
@@ -1022,7 +1022,7 @@ func (consumeOpts *consumeOpts) setDefaults(ordered bool) error {
 		}
 	}
 	if consumeOpts.Heartbeat > consumeOpts.Expires/2 {
-		return errors.New("the value of Heartbeat must be less than 50%% of expiry")
+		return fmt.Errorf("%w: the value of Heartbeat must be less than 50%% of expiry", ErrInvalidOption)
```

Comment:
```suggestion
		return fmt.Errorf("%w: the value of Heartbeat must be less than 50% of expiry", ErrInvalidOption)
```

Reply:
Well actually that's correct, since you have to escape `%` with another `%` when formatting a string 🤷